src/js/components/Button/doc.js

```diff
@@ -66,7 +66,9 @@ with plain Buttons.`,
 causes the Button to be disabled.`,
     ),
     plain: PropTypes.bool
-      .description('Whether this is a plain button with no border or padding.')
+      .description(
+        'Whether this is a plain button with no border or pad. non plain button will show both padd and border. plain button has no border, and unless the icon prop exist it has no pad as well.',
```

Comment:
This line needs to be split into multiple lines. Also, each sentence should be begin with a capital letter.